frontend/compositions/visualizations/map/marker-layer.tsx

```diff
@@ -1,41 +1,140 @@
+import { select } from "d3"
 import { Coord } from "../utilities/chartTypes"
 import styles from "./map.module.css"
 
 export interface MarkerDescription {
   geoCenter: Coord
-  dataPoint: number
+  dataPoint?: number
   label: string
 }
 
 export interface MarkerLayerProps {
   markersData: MarkerDescription[]
+  type?: "circle" | "square"
+  fill?: string
 }
 
 export function MarkerLayer(props: MarkerLayerProps) {
   const { markersData } = props
+
+  const handleMouseOver = (event: MouseEvent, markerDescription: MarkerDescription) => {
+    console.log("moue enter marker")
+
+    const x = markerDescription.geoCenter.x + (event.target as HTMLElement).offsetLeft
+    const y = markerDescription.geoCenter.y + (event.target as HTMLElement).offsetTop
+
+    // popup.moveTo({x: x, y: y})
+    // popup.setHeader(markerDescription.label)
+    // popup.setBody("this is a marker popup")
+    // popup.show()
+  }
+
+  const handleMouseMove = (event: MouseEvent) => {
+    event.stopPropagation()
+  }
+
+  const handleMouseLeave = () => {
```

Comment:
I'd say remove the console.log/commented out code which leads me to - do we need this method?